setup.py

```diff
@@ -1331,15 +1331,9 @@ def main():
             ]
         )
     torchgen_package_data = [
-        # Recursive glob doesn't work in setup.py,
-        # https://github.com/pypa/setuptools/issues/1806
-        # To make this robust we should replace it with some code that
-        # returns a list of everything under packaged/
-        "packaged/ATen/*",
-        "packaged/ATen/native/*",
-        "packaged/ATen/templates/*",
-        "packaged/autograd/*",
-        "packaged/autograd/templates/*",
+        "packaged/**/*.cpp",
+        "packaged/**/*.h",
+        "packaged/**/*.yaml",
```

Comment:
Is this feature limited only to certain versions of python?

Reply:
This requires `setuptools` to have version >= 62.3.0, which implies needing a python version >= 3.7

Reply:
Sorry, where did you find that statement?